src/openapi/transport/batch.ts

```diff
@@ -117,7 +117,9 @@ class TransportBatch extends TransportQueue {
     ) => {
         // Previously occurred due to a bug in the auth transport
         if (!(batchResult && batchResult.response)) {
-            log.error('Received success call without response', batchResult);

```

Comment:
Thanks to improved EventEmitter types, TS was able to find this bug. `batchResult` was logged as [object object] in Kibana.